pipeline/detection.py

```diff
@@ -624,6 +674,7 @@ def _run_psfex( self, tempname, image, psf_size=None, do_not_cleanup=False ):
                                 '-WRITE_XML', 'Y',
                                 '-XML_NAME', psfxmlfile,
                                 '-XML_URL', 'file:///usr/share/psfex/psfex.xsl',
```

Comment:
...we might want to make this one configurable.  The default will work inside the Docker image, so I don't think this is very urgent.  But, if somebody is trying to run on their own installation without using the Docker image, they may  not have the freedom to put stuff in /usr/share.

Reply:
Maybe this should point to the local directory?

Reply:
Ok I see, this is a configuration file. I opened an issue to deal with this someday.